### ISSUE TITLE
Configuration of LMDB map size in other DBs than Baser

### ISSUE THREAD

**Issue by rodolfomiranda** [WebOfTrust/keripy#853]
### Feature request description/rationale

In PR https://github.com/WebOfTrust/keripy/pull/810, @pfeairheller added a feature to set the maximum file size for the main KERIpy LMDB database (Baser).
In witnesses, we realized that also the mailbox DB can grow considerably, so we need a way to change the `map_size` config for other dbs.
Should we allow the configuration on some selected DBs? or should we apply a global setting in LMDBer? I'd like to hear opinions before starting a PR.

**Comment by SmithSamuelM** [comment]
I suggest that adding parameters to LMDBer.reopen to override the default settings for maxdbs and maxsize will work for all instances. then fix config to provide those overrides when setting up dbs.

**Comment by kentbull** [comment]
This is a duplicate of #988. https://github.com/WebOfTrust/keripy/pull/1012 on `v1.2.6` and https://github.com/WebOfTrust/keripy/pull/1013 on `main` set a pattern we might like.

There is both a global `lmdber` config block and the subclass specific blocks shown below illustrate how to configure map sizes either globally or per class.

```hjson
{
  "dt": "2022-01-20T12:57:59.823350+00:00",
  "lmdber": {
    // LMDB Map Size - 50 MB
    "mapSize": 52428800
  },
  "baser": {
    // LMDB Map size - 1 GB
    "mapSize": 1073741824
  },
  "keeper": {
    // LMDB Map Size - 25 MB
    "mapSize": 26214400
  },
  "mailboxer": {
    // LMDB Map Size - 16.384 GB
    "mapSize": 17179869184
  },
  "reger": {
    // LMDB Map Size - 512 MB
    "mapSize": 536870912
  },
  "noter": {
    // LMDB Map Size - 128 MB
    "mapSize": 134217728
  },
  "wan": {
    "dt": "2022-01-20T12:57:59.823350+00:00",
    "curls": ["tcp://127.0.0.1:5632/", "http://127.0.0.1:5642/"]
  },
  "iurls": [
  ]
}
```

Closing in favor of #988.
